iib/workers/tasks/utils.py

```diff
@@ -25,6 +28,35 @@ def get_image_labels(pull_spec):
     return skopeo_inspect(full_pull_spec).get('Labels', {})
 
 
+def retry(
+    retries=worker_config.worker_total_retries, wait_on=Exception, logger=None,
```

Comment:
If `retries` is 5, I would expect the function to be executed a maximum of 6 times (1 attempt + 5 retries). In the code below, it will only execute it 5 times.

Maybe you can change `retries` to `attempts` to make this less confusing.